products/pdt/pdtdb/src/values/transaction.rs

```diff
@@ -274,6 +275,31 @@ impl BQTransaction {
             eth_transaction_type,
         })
     }
+
+    pub fn from_eth_with_zil_txn_bodies(
```

Comment:
compose new function that merges the 2. This is to ensure previous code does not break. Will refactor the rest once this is done